### PR TITLE
adding draft CONP FAQ file

### PR DESCRIPTION
**Pull request by emmetaobrien**
This PR adds a set of FAQs for the CONP portal; I have tried to be short and punchy and link to existing resources where possible. This is very much a first draft.

Two issues still outstanding here:

1) Should we explicitly mention in this context that DataLad is generating links to external resources rather than hosting them locally, and do we want to offer a local hosting option?

2) We should definitely have something about open science, restricted access and ORCIDs. I am not sure how best to phrase that to accurately convey what we are providing, so would particularly welcome input on this front.

### CODE REVIEW

File: CONP_FAQ.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+4. **How do I download data from CONP?**
+
+Results pages for the individual datasets contain instructions for downloading that dataset.  At the moment, access to datasets is made available through the [DataLad](www.datalad.org) data management system.  Download at the push of a button is under development and will be available shortly.
+
+5. **How do I upload data to CONP?**
+
+Data can be uploaded to CONP either through Zenodo, by tagging your dataset with the keyword 'canadian-open-neuroscience-platform', or manually through DataLad.  [Detailed technical instructions](https://github.com/CONP-PCNO/conp-documentation/blob/master/datalad_dataset_addition_procedure.md) are available for both these options.
+
+6. **Why should I upload data to CONP?**
```
Comment (cmadjar) [inline CONP_FAQ.md]: I would tend to have question 6. before question 5. The `why` before the `how`

File: CONP_FAQ.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+CONP portal FAQ
```
Comment (cmadjar) [inline CONP_FAQ.md]: maybe format `CONP portal FAQ` as a title?

**Comment by emmetaobrien** [comment]
Thanks @cmadjar, that's very helpful.  I'm also not familiar enough with the tools and pipelines to give useful answers for that part of the FAQ, but I have made all the other changes you suggested.

File: CONP_FAQ.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Results pages for the individual datasets contain instructions for downloading that dataset.  At the moment, access to datasets is made available through the [DataLad](www.datalad.org) data management system.  Data download at the push of a button is under development and will be available shortly.
+
+4. **What software do I need to download data from CONP?**
+
+At the moment downloading CONP data requires DataLad and associated software running on a Linux machine ([installation details]](https://github.com/CONP-PCNO/conp-documentation/blob/master/datalad_dataset_addition_procedure.md)). Download through the browser will be available shortly.
+
+5. **I clicked on the download button on the file list, where is the data?**
+
+The download buttons on the list of files downloads metadata for each dataset, in a single .json file which contains a detailed description of the data in DATS format.  The data files themselves can be downloaded using DataLad as described on the results page for each individual dataset.
```
Comment (cmadjar) [inline CONP_FAQ.md]:
```suggestion
The download buttons on the list of files downloads metadata for each dataset, in a single `.json` file which contains a detailed description of the data in DATS format.  The data files themselves can be downloaded using DataLad as described on the results page for each individual dataset.
```

File: CONP_FAQ.md (hunk 1 above)
Comment (cmadjar) [reply CONP_FAQ.md]: (resolved)

**Comment by cmadjar** [comment]
@glatard Could you add some FAQ questions regarding tools and pipelines? Or assign that to someone who could create those FAQ questions? Thank you!

**Comment by cmadjar** [comment]
@emmetaobrien should we just merge this first FAQ document and then ask people to add questions for the pipeline sections?

**Comment by emmetaobrien** [comment]
At this point I think so, yes.

**Pull request merged**